content/2016-12-26_komunitni-python-stanek-na-linuxdays.md

```diff
@@ -0,0 +1,90 @@
+Title: Komunitní Python stánek na LinuxDays
+Date: 2016-12-26 13:28:00
+Modified: 2016-12-26 13:28:00
+Author: Veronika Gabrielová
+Twitter: lspdv
+Gravatar: gabrielova.veronika@gmail.com
+About: Veronička je velmi aktivní v české [Python](http://python.cz/) komunitě. Motá se hodně kolem [PyLadies](http://pyladies.cz/), účastnicím po kurzech pomáhá najít vysněnou práci a právě buduje rozjezd i dalších navazovacích kurzů a srazů pro všechny zájemce o Python, kteří jsou v začátcích. Také již pracuje jako programátorka. Nemá ráda datle. Má ráda vše ostatní kromě datlí. Nejvíc Žufánka a Pho. Nejčastěji ji potkáš v Praze na Pyvo srazech či na PyWorkingu a PyLadies.
+
+### Organizátoři LinuxDays nás oslovili, zdali chceme využít možnosti prezentovat se na známé akci LinuxDays. 
+
+```
```

Comment:
Nevím, chtěla jsem aby to bylo prostě pozadím rozlišené. Jestli máš jiný nápad, sem s ním.